ONSdigital/dp-census-atlas
# Feature/make validate content.ts from csv + update content.ts with V4_3 content

### What

commit 5d47e705d520f7d0af9288d651afb1554308ff46 (HEAD -> feature/make-validate-content.ts-from-csv, origin/feature/make-validate-content.ts-from-csv)
Author: Vivian Allen <vivian.allen@methods.co.uk>
Date:   Thu Mar 17 22:04:34 2022 +0000

    curation.ts -> content.ts for consistency, v4_3 content added

    Rename curation.ts to content.ts to match more generally-used
    term

    Update content.ts with latest V4_3 content from:
    https://docs.google.com/spreadsheets/d/1AD0LGGfPEO2vDRBYWxoEEAf5fjILV9TaUrctHexyhT0/edit#gid=560262497

commit feb2b93d95b041fee62a4c922fd2976b57d50a1a
Author: Vivian Allen <vivian.allen@methods.co.uk>
Date:   Thu Mar 17 22:01:55 2022 +0000

    content-csv-to-json makes atlas content JSON from csv files

    content-csv-to-json dir added with following:

    - content_csv_to_json.py (python3): converts csv files with content
    (data labels, descriptive text etc) downloaded from spreadsheet
    shared with design team to JSON consumable by atlas.

    - validate_content_json.oy (python3): Checks that all census data
    referenced in the output of content_csv_to_json.py is available from
    the dp-geodata-api.

    These changed needed to streamline getting content from design team
    to front end.

### How to review

- make sure all references to curation.ts chagned to content.ts (unless reviewers feel this is an uneccessary change)
- make sure the deployed preview functions properly with the new content
- try out the content CSV -> content JSON python parse in content-csv-to-json/ (instructions in README) with a content csv downloaded from V4_3 here: 
https://docs.google.com/spreadsheets/d/1AD0LGGfPEO2vDRBYWxoEEAf5fjILV9TaUrctHexyhT0/edit#gid=560262497
- see what you think about the new 'data percentage component' template strings in the category objects in content.ts - I've got them down as 'category_h_ptX' for 'category header', but not married to the name

### Who can review

Anyone
